src/components/map/mapControls/mapLayersControl.scss

```diff
@@ -24,6 +25,31 @@
         border: 1px solid $darkGrey;
         z-index: 1;
         padding: 10px;
+
```

Comment:
There's a non-breaking space after `.mapLayersContainer`. Seems like it's been there before, but I think it should be removed asap.

Reply:
Tried removing it. Didn't work. Added it back.